src/setupTools.js

```diff
@@ -217,6 +217,15 @@ export default function setupTools(store) {
     { name: 'ZoomTouchPinch' },
     { name: 'StackScrollMouseWheel' },
     { name: 'StackScrollMultiTouch' },
+    {
+      name: 'Eraser',
+      mouseButtonMasks: [1],
+      props: {
```

Comment:
I don't think the Eraser tool should have these options

Reply:
You're absolutely right. Looks like I added this while I was trying to resolve a different issue that turned out to be caused by something in toolbarModule.js, which isn't included in this review. I'll get a fix.

Thanks Erik!

Reply:
I also removed the mouseButtonMasks this morning. Turns out that wasn't need either.